pybible/__main__.py

```diff
@@ -71,6 +74,12 @@ def process_arguments(parser: argparse.ArgumentParser):
         else:
             references = [Reference(verse.text, book.title, chapter.number, verse.number)]
 
+    elif args.qotd:
+        book = random.choice(bible)
+        chapter = random.choice(book.chapters)
+        verse = random.choice(chapter.verses)
+        return f"\"{verse.text}\" - {book.title} {chapter.number}:{verse.number} ({bible.name})"
```

Comment:
replace with:
`references = [Reference(verse.text, book.title, chapter.number, verse.number)]`
too keep the new parameter the same as the others.